src/twisted/web/http.py

```diff
@@ -733,8 +733,8 @@ class Request:
     finished = 0
     code = OK
     code_message = RESPONSES[OK]
-    method = "(no method yet)"
-    clientproto = b"(no clientproto yet)"
+    method = b"(no method yet)"  # type: bytes
```

Comment:
why is this required? should this be auto detected from the normal assignment ?
Is this a mypy bug?

Reply:
All subclasses define method as bytes, which is correct.  It is incorrectly defined as str here.

For example, elsewhere in this file:

```
        if self.method == b"POST" and ctype and clength:
            mfd = b'multipart/form-data'
            key, pdict = _parseHeader(ctype)
```